packages/sdk/src/realtime/client.ts

```diff
@@ -45,12 +66,19 @@ export type RealTimeClient = {
   on: <K extends keyof Events>(event: K, listener: (data: Events[K]) => void) => void;
   off: <K extends keyof Events>(event: K, listener: (data: Events[K]) => void) => void;
   sessionId: string;
+  // Avatar-live audio methods (only available when model is avatar-live)
+  playAudio?: (audio: Blob | File | ArrayBuffer) => Promise<void>;
+  stopAudio?: () => void;
+  isPlayingAudio?: () => boolean;
 };
 
 export const createRealTimeClient = (opts: RealTimeClientOptions) => {
   const { baseUrl, apiKey, integration } = opts;
 
-  const connect = async (stream: MediaStream, options: RealTimeClientConnectOptions): Promise<RealTimeClient> => {
+  const connect = async (
+    stream: MediaStream | null,
```

Comment:
I suspect that not passing `stream` diverges a bit too much from the original purpose of this method?

It looks like we're handling the `userAudio || silence` case end-to-end - but should we be? Does this actually help users, or could it end up limiting them?

e.g. can devs wire the end-user's mic directly now, to have the avatar say whatever they say? I think this might be a common use-case.